test/secrets/ocsp_responder_redis_password

```diff
@@ -0,0 +1 @@
+0e5a4c8b5faaf3194c8ad83c3dd9a0dd8a75982b
```

Comment:
Missing a newline here. I just checked and `cmd.Config.PasswordConfig.Pass()` will trim it for you on load too.

```suggestion
0e5a4c8b5faaf3194c8ad83c3dd9a0dd8a75982b

```

Reply:
Fixed, although you might have a hard time verifying unless you try to "edit" the file. Github doesn't want to show the blank newline. Thanks!

Reply:
I can tell that it's fixed because when you don't include a newline GH displays a little angry red circle with a minus line, which is how I noticed it was missing.